mindmeld/components/entity_resolver.py

```diff
@@ -51,51 +65,189 @@ class EntityResolver:
     (usually linked to specific entries in a knowledge base).
     """
 
-    # prefix for Elasticsearch indices used to store synonyms for entity resolution
-    ES_SYNONYM_INDEX_PREFIX = "synonym"
-    """The prefix of the ES index."""
-
-    def __init__(
-        self, app_path, resource_loader, entity_type, es_host=None, es_client=None
-    ):
-        """Initializes an entity resolver
+    @classmethod
```

Comment:
I am not convinced why we need to use a classmethod here instead of an instance method.

Reply:
Agreed. I think it can be `@staticmethod` instead of `@classmethod`. Will correct that!